tests/e2e/provisioning_e2e/tests/test_sync_symmetric_enrollments.py

```diff
@@ -6,31 +6,28 @@
 
 from provisioning_e2e.service_helper import Helper, connection_string_to_hostname
 from azure.iot.device import ProvisioningDeviceClient
-from azure.iot.device.common import X509
-from provisioningserviceclient import ProvisioningServiceClient, IndividualEnrollment
-from provisioningserviceclient.protocol.models import AttestationMechanism, ReprovisionPolicy
+from ..provisioningservice.protocol.models import (
+    IndividualEnrollment,
+    AttestationMechanism,
+    ReprovisionPolicy,
+    ClientCertificateIssuancePolicy,
+)
+from ..provisioningservice.client import ProvisioningServiceClient
 import pytest
 import logging
 import os
 import uuid
-from . import path_adjust  # noqa: F401
-from create_x509_chain_crypto import (
-    create_private_key,
-    create_csr,
-)
-from azure.iot.device import IoTHubDeviceClient
 
-logging.basicConfig(level=logging.DEBUG)
 
 PROVISIONING_HOST = os.getenv("PROVISIONING_DEVICE_ENDPOINT")
 ID_SCOPE = os.getenv("PROVISIONING_DEVICE_IDSCOPE")
 service_client = ProvisioningServiceClient.create_from_connection_string(
     os.getenv("PROVISIONING_SERVICE_CONNECTION_STRING")
 )
-device_registry_helper = Helper(os.getenv("IOTHUB_CONNECTION_STRING"))
-linked_iot_hub = connection_string_to_hostname(os.getenv("IOTHUB_CONNECTION_STRING"))
-# TODO Delete this line. This is a pre created variable in key vault now.
-symmetric_key_for_cert_management = os.getenv("DPS_CERT_ISSUANCE_SYM_KEY_SYNC")
+connection_string = os.getenv("IOTHUB_CONNECTION_STRING")
```

Comment:
Outside the scope of this PR, but at some point we should probably try and overhaul these DPS E2Es to use some of the enhancements and shared infrastructure Bert created for IoTHub E2Es + stress tests.

Probably can wait til V3 though